percolator/challenges/default.json

```diff
@@ -99,33 +118,4 @@
           "target-throughput": 50
         }
       ]
-    },
-    {
-      "name": "append-no-conflicts-index-only",
```

Comment:
We don't need this challenge any more? I would have assumed we'd just need to replace the operations here with a delete / create / index, but what am I missing?

Reply:
Percolator really is about querying, indexing is just a means to an end here and I figured we can get rid of it right away. If somebody really wants this (which they shouldn't), then they can just use task filtering to get it back.